### PR TITLE
Update master to 1.1.2

### PR DESCRIPTION
**Pull request by NicoleEO**
Includes DE, bug fixes, and documentation updates

### CODE REVIEW

**Comment by NicoleEO** [comment]
@rvitancol this includes review of all these commits https://github.com/Nanostring-Biostats/GeomxTools/compare/8b9db0aa382c3829189c4b391655d0926d420ce0...dev

**Review by rvitancol — CHANGES_REQUESTED** [review]

File: tests/testthat/test_GeoMxSet_probeCollapse.R [hunk 1]
```diff
@@ -1,81 +1,72 @@
...
+                    length(unique(fData(subData)[["TargetName"]])))
+    subList <- lapply(featureNames(subSum), function(currTarg) {
+        aggdCounts <- exprs(subSum)[currTarg, sampleNames(subData)]
+        targData <- exprs(subset(subData, subset=TargetName == currTarg))
+        targSums <- apply(targData, 2L, function(sampCount) {
+            ifelse(length(sampCount) > 1, sum(sampCount), sampCount)})
+        return(all(aggdCounts == targSums))
+    })
+    expect_true(all(unlist(subList)))
+})
```
[the remark this answers is not in the document]
Comment (NicoleEO) [reply tests/testthat/test_GeoMxSet_probeCollapse.R:72]: We will add these in a separate PR. @maddygriz is working on this issue now where I logged the above. https://github.com/Nanostring-Biostats/GeomxTools/issues/107

File: R/NanoStringGeoMxSet-aggregate.R [hunk 2]
```diff
@@ -16,9 +16,41 @@
...
+    
+    targetCounts <- do.call(rbind, esBy(multiObject, GROUP = "TargetName", 
         FUN=function(x) {esApply(x, 2, FUN)}, simplify=FALSE))
+    singleProbeObject <- subset(object, 
+                                subset=!TargetName %in% multiProbeTargs)
+    singleProbeCounts <- exprs(singleProbeObject)
+    rownames(singleProbeCounts) <- fData(singleProbeObject)[["TargetName"]]
+    targetCounts <- rbind(targetCounts, singleProbeCounts)
+    targetCounts <- targetCounts[unique(fData(object)[["TargetName"]]), ]
+
```
[the remark this answers is not in the document]
Comment (NicoleEO) [reply R/NanoStringGeoMxSet-aggregate.R:53]: Fixed

File: R/NanoStringGeoMxSet-aggregate.R [hunk 3]
```diff
@@ -29,6 +61,7 @@ aggregateCounts <- function(object, FUN=ngeoMean) {
     targetFeats <- 
          AnnotatedDataFrame(targetFeats[rownames(targetCounts), ], 
                             dimLabels = c("featureNames", "featureColumns"))
+
```
[the remark this answers is not in the document]
Comment (NicoleEO) [reply R/NanoStringGeoMxSet-aggregate.R:65]: Fixed

**Review by rvitancol — APPROVED** [review]

**Pull request merged**